docs/modules/configuration/pages/dynamic-config-rest.adoc

```diff
@@ -0,0 +1,160 @@
+= Adding Dynamic Configuration in REST
```

Comment:
1. You send **exactly one** declarative config as string.
2. Dynamically addable sub-configs will be added cluster one by one.
3. If any one of them **conflicts** with the sub config in **runtime config** (config object) you will get exception. Else it will succeed.

There is nothing related to files. It doesn't matter whatever you do with them if you use `update`.

Reply:
Maybe we can add something like these to make it clearer.

Reply:
I added some clarifications that might help

Reply:
@frant-hartm do the changes make it clearer?